stored_requests/config/config.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // This gets set to the connection string used when a database connection is made. We only support a single
 // database currently, so all fetchers need to share the same db connection for now.
-var dbConnection struct {
+type dbConnection struct {
```

Comment:
Two thoughts:
(1) Comment above the type declaration seems a little inconsistent, though I'm not sure what to suggest.
(2) Why eliminate the global variable?  The new approach seems to allow for future possible programming errors if NewStoredRequests() should somehow be called twice.  In that case, I believe the first call to CreateStoredRequests() in the second call to NewStoredRequests() would pass an uninitialized dbc value leading to a second DB connection attempt even though one already exists.  That danger doesn't seem to exist with the current global variable in place.  As this code is written, I don't think two calls to NewStoredRequests() can occur, though, so this is more a precautionary suggestion.

Reply:
Global variables are bad. :) The way it was, it was mostly fine, but could potentially cause issues for testing if multiple test threads where running where the tests all assumed they were the only ones messing with `dbConnection` things.

With this change, we are 90% of the way to supporting multiple databases. Yes, a second call to `NewStoredRequests()` would create a second valid database connect, a second set of fetchers, and a second shutdown function. This should all be fine if you treat them as a second set of these things that need to be managed, but of course bad if you then conflate the members of the two sets.

I feel that getting rid of the mutable global state is worth the risk of accidentally calling `NewStoredRequests()` twice and not handling the results properly.